cryptography/src/bloomfilter.rs

```diff
@@ -20,6 +19,9 @@ const HALF_DIGEST_LEN: usize = 16;
 /// The length of a full [Digest].
 const FULL_DIGEST_LEN: usize = Digest::SIZE;
 
+const BITMAP_CHUNK_SIZE: usize = 1;
```

Comment:
The chunk size was 1 before so I kept it. We could change this to a larger value if we want. Same goes for other files.